src/features/charts/redux/middleware.ts

```diff
@@ -0,0 +1,16 @@
+import { Middleware } from 'redux'
+import { UPDATE_CHART_RANGE } from '../redux'
+import { AppState, MiddlewareDependencies } from 'models'
+
+export const chartMiddleware = ({socket}: MiddlewareDependencies): Middleware<AppState> => {
+    return ({getState}) => {
+        return (next) => (action) => {
+            if (action.type === UPDATE_CHART_RANGE) {
+                const { search: ticker } = getState();
```

Comment:
I'd find `const ticket = getState().search` easier to read

Reply:
I think I have a few place where destructuring has occured but isn't entirely necessary.

I've made this change but will fix others in future PR.

I've raised this as an issue.